packages/neon-domain/__integration__/main.ts

```diff
@@ -0,0 +1,51 @@
+import apiPlugin from "@cityofzion/neon-api";
+import * as neonCore from "@cityofzion/neon-core";
+import domainPlugin from "../src/index";
+
+const neonWithDomain = apiPlugin(neonCore);
+const neonJs = domainPlugin(neonWithDomain);
+const { CONST, rpc, u, api, wallet } = neonJs;
+
+let MAINNET_URL = "";
+let provider;
+
+beforeAll(async () => {
+  provider = new neonJs.domain.nns.instance("348387116c4a75e420663277d9c02049907128c7");
+  try {
+    const testUrl = "http://seed1.cryptoholics.cc:10332";
+    const config = {
+      api: {
+          getRPCEndpoint: jest.fn().mockImplementationOnce(() => testUrl)
+      } as any,
+          address: ""
+    }
+      const result = await neonJs.api.fillUrl(config);
```

Comment:
I don't understand why are we depending on the api plugin for test setup but mocking it out. This doesn't really serve any purpose. 

Do look at the integration test for nep5. I have a simple loop that attempts to get a good rpc node to test again.